### PR TITLE
Changed the namespace for all the Issues in the control gallery to *Xamarin.Forms.Controls.Issues*

### LINKED ISSUE #4150: Change the namespace for all the Issues in the control gallery to *Xamarin.Forms.Controls.Issues*
### Description
If a UI Test isn't in the *Xamarin.Forms.Controls.Issues* namespace and you run that as the first UI Test then this code doesn't run

```
#if UITEST
namespace Xamarin.Forms.Controls.Issues
{
	using System;
	using NUnit.Framework;

	// Run setup once for all tests in the Xamarin.Forms.Controls.Issues namespace
	// (instead of once for each test)
	[SetUpFixture]
	public class IssuesSetup
	{
		[SetUp]
		public void RunBeforeAnyTests()
		{
			AppSetup.RunningApp = AppSetup.Setup(null);
		}
	}
}
#endif
```

And you get a Null Reference Exception
https://github.com/xamarin/Xamarin.Forms/pull/4144#discussion_r226783148

### Steps to Reproduce

1.  Find a UI Test
2.  change the namespace to namespace Xamarin.Forms.Controls
3. Try to run the UI Test

### Expected Behavior
Make it easier for contributors to add UI Tests to existing issues and not have them fail

### PR DESCRIPTION
**Pull request by ionixjunior**
### Description of Change ###

Changed namespaces of all UI tests different of **Xamarin.Forms.Controls.Issues**.

During the refactoring of the namespaces some problems was found in compilation. It was necessary move some helper classes that were inside UI tests to another structure using **Xamarin.Forms.Controls** namespace. I created a new folder called **Controls** with all this refactoring.

### Issues Resolved ### 

- fixes #4150

### API Changes ###

None

### Platforms Affected ### 

All platforms that contains UI tests.

### Behavioral/Visual Changes ###

None

### Before/After Screenshots ### 

Not applicable

### Testing Procedure ###

Run UI tests on all platforms to check if all works fine. 
In my compilation test, I tested Android, GTK, iOS, MacOS and Tizen compilations of the ControlGallery project. WPF and WindowsUniversal platforms I could not perform the compilation test.

### PR Checklist ###

- [x] Has automated tests
- [x] Rebased on top of the target branch at time of PR
- [x] Changes adhere to coding standard

### CODE REVIEW

**Comment by ionixjunior** [comment]
Rebase performed again.

File: Xamarin.Forms.Controls.Issues/Xamarin.Forms.Controls.Issues.Shared/Bugzilla42602.cs [hunk 1]
```diff
@@ -7,7 +7,7 @@
 using Xamarin.Forms.Core.UITests;
 #endif
 
-namespace Xamarin.Forms.Controls
+namespace Xamarin.Forms.Controls.Issues
```
Comment (hartez) [inline Xamarin.Forms.Controls.Issues/Xamarin.Forms.Controls.Issues.Shared/Bugzilla42602.cs]: You'll need to update this namespace in ExportRenderer attribute in Xamarin.Forms.ControlGallery.WindowsUniversal.CustomRenderers.cs.

**Comment by rmarinho** [comment]
Test 2951 is failing

**Comment by hartez** [comment]
I think the issue is that `ButtonExtensions` is no longer in the `Xamarin.Forms.Controls` namespace after this change. You can probably fix this by adding another namespace declaration to Issue2951.xaml, something like:
`xmlns:extensions="clr-namespace:Xamarin.Forms.Controls.Issues;assembly=Xamarin.Forms.Controls"`

And then changing all the instances of `controls:ButtonExtensions` to `extensions:ButtonExtensions`.

**Comment by ionixjunior** [comment]
@hartez yes, in this case it makes more sense.

Similar situations occurred when I finished the first refactoring, but I extract internal implementations to a new folder with old namespace (bacause some implementations where used in another tests/projects).

I will follow your suggestion and pull request again.

**Comment by ionixjunior** [comment]
Done. Fixed namespace reference in Issue2951 test.

**Comment by ionixjunior** [comment]
Solved.

**Comment by ionixjunior** [comment]
Only Android API25 Validation Fast Renderers broken. This is normal?

It’s not the first time that happens.

**Pull request merged**